src/UI/Component/DropDown/DropDownItem.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/* Copyright (c) 2017 Alexander Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\DropDown;
+
+use ILIAS\UI\Component\Clickable;
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Hoverable;
+use ILIAS\UI\Component\JavaScriptBindable;
+
+/**
+ * Drop down items
+ */
+interface DropDownItem extends Component, JavaScriptBindable, Clickable, Hoverable {
```

Comment:
Or we have DividerItem and LinkItem being derived from DropdownItem and a DropDown constructor with (DropDownItems[] $items) being passed? Or a Link and Divider on a higher level implementing a DropdownItem interface?

I think we definitely will need the signal/ID magic on the items.